src/screens/Manager/AppsList/AppFilter.js

```diff
@@ -0,0 +1,70 @@
+import React, { memo, useState, useCallback, useMemo } from "react";
+import { StyleSheet, View } from "react-native";
+
+import Filters from "../../../icons/Filters";
+import Button from "../../../components/Button";
+import NotifBadge from "../../../components/NotifBadge";
+
+import FilterModalComponent from "../Modals/FilterModal";
+
+type Props = {
+  filter: string,
+  setFilter: Function,
+  sort: string,
+  setSort: Function,
+  order: string,
+  setOrder: Function,
+  disabled: Boolean,
+};
+
+const AppFilter = ({
+  filter,
+  setFilter,
+  sort,
+  setSort,
+  order,
+  setOrder,
+  disabled,
+}: Props) => {
+  const [isOpened, openModal] = useState(false);
+  const toggleModal = useCallback(value => () => openModal(value), [openModal]);
```

Comment:
the `useCallback` usage can be either removed or split into two functions (one for true, one for false)
because the fact it's a `()=>()=>..` makes the `useCallback` useless